# HG changeset patch
# User Rik <rik@octave.org>
# Date 1398972890 25200
#      Thu May 01 12:34:50 2014 -0700
# Node ID c7969670146870bdd7b4927fc90ad3283be876dd
# Parent  0585787aa8ae0bd8a5b1eaa22af96affe7b92f90
doc: Document new 'colormap' parameter to imshow.

* imshow.m: Document new 'colormap' parameter to imshow.

diff --git a/scripts/image/imshow.m b/scripts/image/imshow.m
--- a/scripts/image/imshow.m
+++ b/scripts/image/imshow.m
@@ -40,16 +40,19 @@
 ## shown.
 ##
 ## If given, the parameter @var{string_param1} has value
 ## @var{value1}.  @var{string_param1} can be any of the following:
 ##
 ## @table @asis
 ## @item @qcode{"displayrange"}
 ## @var{value1} is the display range as described above.
+##
+## @item @qcode{"colormap"}
+## @var{value1} is the colormap to use when displaying an indexed image.
 ## 
 ## @item @qcode{"xdata"}
 ## If @var{value1} is a two element vector, it must contain horizontal axis
 ## limits in the form [xmin xmax]; Otherwise @var{value1} must be a
 ## vector and only the first and last elements will be used for xmin and
 ## xmax respectively.
 ## 
 ## @item @qcode{"ydata"}
